src/containers/Home/styles.scss

```diff
@@ -41,14 +41,36 @@
    
     
     .job__content{
+      position: relative;
       display:flex;
-      
-      width: 300px ; 
-      height: 300px ;
+      width: 300px; 
+      height: 300px;
       background-color: #fff;
       border-radius: 30px;
       flex-direction: column;
       justify-content: center;
-     
+      .content-tag{
+        border-radius: 30px;
+        position: absolute;
+        display: inline-block;
+        width: 100px;
+        height: 100px;
+        z-index: 100;
+        right: 0;
+        top:0;
+        span{
```

Comment:
why you define content__status classes, but i cannot see using instead of span tag ??

Reply:
I have already fixed